autoload/sentencer.vim

```diff
@@ -159,18 +159,26 @@ function! s:split(line, indent1, indent, o) abort
   return l:lines
 endfunction
 
-function! sentencer#Format() abort
+function! sentencer#Format(...) abort
   let l:o = s:options()
   let l:pos = getcurpos()
-  let l:start = v:lnum
-  let l:end = l:start + v:count - 1
+  if a:0 " from :Sentencer command
+      let l:start = a:1
+      let l:end = a:2
+  else " from formatexpr
+      let l:start = v:lnum
+      let l:end = l:start + v:count - 1
+  endif
   let l:orig = getline(l:start, l:end)
 
   let l:lines = []
   for [l:indent1, l:indent, l:blank, l:para] in s:paragraphs(l:orig, l:o)
     if l:blank
       let l:lines += ['']
     endif
+    if len(l:para) == 0
```

Comment:
If you have a specific example in mind that requires this check would you mind adding it as a test?

Reply:
The first element of the list returned by `s:paragraphs` is always `[-1, -1, 0, []]`, which causes "list index out of range" error in `l:para[0]`. But somehow, the error is silently ignored when the function is run from formatexpr.

Reply:
Ah, I see. That's annoying that it suppresses error messages, `indentexpr` does too I think. Then my only suggestion is to use `empty()` instead to check for that case.